platform_versioned_docs/version-24.1.0/enterprise/_templates/k8s/data_studios/proxy.yml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: connect-proxy
+  labels:
+    app.kubernetes.io/component: tower-connect-proxy
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: tower-connect-proxy
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: tower-connect-proxy
+    spec:
+      nodeSelector:
+        kubernetes.io/arch: amd64
+      containers:
+        - name: proxy
+          image: cr.seqera.io/private/nf-tower-enterprise/data-studio/connect-proxy:0.7.0
+          command:
+            - caddy
+            - run
+            - --config
+            - /proxy-config/Caddyfile
+          env:
+            - name: CONNECT_HTTP_PORT
```

Comment:
Environment variable names were updated - see the latest version here: https://github.com/seqeralabs/platform-deployment/commit/8ce70cbcd01209457f2067baf819026a0cc732a3